src/crate/theme/rtd/crate/sections/sidebar-secondary.html

```diff
@@ -13,6 +13,17 @@
       {%- if display_version %}
       {% include "components/version_chooser.html" %}
       {%- endif %}
+
+      {% include "partials/icons.html" %}
```

Comment:
See other comment. Can we include the `icons.html` from a canonical place like before, instead of this specific place in `sidebar-secondary.html`?

Reply:
should I do that before we publish the dev release?

Reply:
moved it back

Reply:
Thanks!